tests/instruments/test_instrument.py

```diff
@@ -317,6 +446,11 @@ class GenericInstrument(FakeInstrument):
     )
 
 
+@pytest.fixture(scope="module")
+def generic():
+    return GenericInstrument()
```

Comment:
Can you move fixtures and such tooling to before the place of first use, please?

Reply:
...or the tests using it after that?